static/app/utils/profiling/renderers/flamegraphTextRenderer.tsx

```diff
@@ -35,7 +35,7 @@ class FlamegraphTextRenderer extends TextRenderer {
   draw(
     configView: Rect,
     configViewToPhysicalSpace: mat3,
-    flamegraphSearchResults: FlamegraphSearch['results']['frames']
+    flamegraphSearchResults?: FlamegraphSearch['results']['frames']
```

Comment:
You could have just passed an Empty Map to the draw call :D 

It's nbd, but the reason I changed that was to avoid having this function having a polymorphic type. (e used to call draw with undefined|map depending on if user was searching or not)